cmd/helm/template.go

```diff
@@ -96,6 +97,7 @@ func newTemplateCmd(out io.Writer) *cobra.Command {
 	f.VarP(&t.valueFiles, "values", "f", "specify values in a YAML file (can specify multiple)")
 	f.StringVar(&t.namespace, "namespace", "", "namespace to install the release into")
 	f.StringArrayVar(&t.values, "set", []string{}, "set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
+	f.StringArrayVar(&t.values, "set-string", []string{}, "set STRING values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
```

Comment:
Looks like a copy/paste error. This should be `t.stringValues`.

Reply:
Nice catch, updated 👍